operator/node.go

```diff
@@ -68,9 +68,23 @@ func New(opts Options) Node {
 			DutyLimit:           opts.DutyLimit,
 		}),
 	}
+
+	if err := ssv.init(opts); err != nil {
+		ssv.logger.Fatal("failed to init", zap.Error(err))
+	}
+
 	return ssv
 }
 
+func (n *operatorNode) init(opts Options) error {
```

Comment:
@amir-blox why not put it in the cli package?

Reply:
Same comment as above:

The delete is not "generic" - it uses some prefix, meaning we need to perform this from the actual collection/storage where the prefix is defined